qt/widgets/common/inc/MantidQtWidgets/Common/PropertyHandler.h

```diff
@@ -177,6 +177,9 @@ class EXPORT_OPT_MANTIDQT_COMMON PropertyHandler
   double height() const;
   double centre() const;
   double fwhm() const;
+  std::string getWidthParameterName() const;
+  std::string getCentreParameterName() const;
+  bool getParameterNameExplicitlySet(const std::string &param) const;
```

Comment:
Same as above: Maybe this should be called `isParameterNameExplicitlySet` or  `isParameterExplicitlySet`, the current name `getParameterNameExplicitlySet` makes it seem like its returning a parameter or parameter name.